### PR TITLE
feat(#17): issue pagination

### PR DESCRIPTION
**Pull request by yoannfleurydev**
related to #17

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -83,7 +83,7 @@ model Scope {
   name        String           @unique
   description String?          @db.Text
   color       String?
-  createdAt   DateTime         @default(now())
+  createdAt   DateTime         @unique @default(now())
```
Comment (ivan-dalmet) [inline prisma/schema.prisma:86]: Why? (I suppose it's for sorting but my guess is to use multi column sorting and not this)

File: src/server/routers/issue.ts [hunk 2]
```diff
@@ -5,6 +5,60 @@ import { z } from 'zod';
...
+        take: limit + 1,
+        where: {
+          title: {
+            search: search !== '' ? search : undefined,
+          },
+          description: {
+            search: search !== '' ? search : undefined,
+          },
+        },
+        cursor: cursor ? { createdAt: cursor } : undefined,
```
Comment (ivan-dalmet) [inline src/server/routers/issue.ts:25]: Cursor should be the ID and not creatdAt no?

File: prisma/schema.prisma (hunk 1 above)
Comment (ivan-dalmet) [reply prisma/schema.prisma:86]: Look at this code for reference https://github.com/ivan-dalmet/react-query-offline-demo/blob/master/pages/api/posts/index.ts 😉

**Review by ivan-dalmet — CHANGES_REQUESTED** [review]

File: prisma/schema.prisma (hunk 1 above)
Comment (yoannfleurydev) [reply prisma/schema.prisma:86]: Yep, I'm thinking about adding a new field as the id is not autoincremented

File: src/server/routers/issue.ts (hunk 2 above)
Comment (yoannfleurydev) [reply src/server/routers/issue.ts:25]: Cursor should be a unique and sortable.

File: prisma/schema.prisma (hunk 1 above)
Comment (ivan-dalmet) [reply prisma/schema.prisma:86]: In the example the ID is a uuid, I don't think you need another field (or maybe I missing something)

Comment (yoannfleurydev) [reply prisma/schema.prisma]: I changed for the `id` as the cursor. At the moment, I didn't change it to an integer as it seems to work. The documentation is quite evasive on the subjet.

**Review by ivan-dalmet — APPROVED** [review]

**Pull request merged**